### PR TITLE
Rename '1D profile' methods

### PR DESCRIPTION
**Pull request by anowacki**
Previously, we had a `get_` method which returned the radial or 1D
profile of a model at a certain point, but `get_` methods are by
convention those which return the data inside a model, possibly
for editing.  In updating this name, a simple change to
`1d_profile` was not possible under Python's naming rule so another
name was needed.  This commit changes `TerraModel.get_1d_profile` to
`TerraModel.radial_profile`, and for consistency also changes
`TerraModel.mean_1d_profile` to `TerraModel.mean_radial_profile`.

As an alternative I considered `profile_1d`, but that didn't feel
right.  A '1D profile' is really tautologous, since a profile is by
definition 1D.  Hence adding information about in which direction
the profile was running seemed most useful.  Finally therefore we
settle on `TerraModel.radial_profile` as the best alternative.

Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. 

*Describe what you did in this PR and why you did it.*

### For all pull requests:

* [x] I have run the [`contrib/utilities/indent`](../blob/main/contrib/utilities/indent) script from the main directory to indent my code.

### For new features/models or changes of existing features:

* [x] I have tested my new feature locally to ensure it is correct.
* [x] I have created a testcase for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.

### CODE REVIEW

**Comment by anowacki** [comment]
@jamespanton93 I've opened this as a request for feedback.  If you're happy with the changes, then I think our API is now pretty much ready.

**Comment by anowacki** [comment]
Docs build seems to have failed for an unrelated reason.

File: terratools/terra_model.py [hunk 1]
```diff
@@ -885,32 +885,46 @@ def mean_1d_profile(self, field):
...
+        Return the radial profile of the given field
+        at a given longitude and latitude point.
 
-        :param field: name of field.
+        :param field: Name of field.
         :type field: str
 
-        :param field: latitude to get 1d profile.
-        :type field: float
+        :param lat: Longitude at which to get radial profile.
```
[the remark this answers is not in the document]
Comment (anowacki) [reply terratools/terra_model.py:893]: Ack, thought I'd sorted that out.  Well spotted.

[the remark this answers is not in the document]
Comment (anowacki) [reply terratools/terra_model.py:896]: Sorted out in the latest commit.

**Review by jamespanton93 — APPROVED** [review]
👍

**Pull request merged**